tox.ini

```diff
@@ -71,35 +75,14 @@ commands =
     echo "Starting {envname}"
     {envpython} -V
     {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_1} --pattern "{[local_settings]test_pattern}"
-    {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_2} --pattern "{[local_settings]test_pattern}"
-    {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_3} --pattern "{[local_settings]test_pattern}"
-    {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_4} --pattern "{[local_settings]test_pattern}"
-    echo "Completed {envname}"
-
-[testenv:py3{9,10,11,12}]
-description = 'Run unit tests (unittest runner) using {envpython}'
-platform=
-       macos: darwin
-       linux: linux
-skip_install = false
-skipsdist = false
-recreate = true
-alwayscopy=true
-usedevelop=true
-deps = 
-       -r requirements.txt
-commands =
-    echo "Starting {envname}"
-    {envpython} -V
-    {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_1} --pattern "{[local_settings]test_pattern}"
-    {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_2} --pattern "{[local_settings]test_pattern}"
-    {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_3} --pattern "{[local_settings]test_pattern}"
-    {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_4} --pattern "{[local_settings]test_pattern}"
+#    {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_2} --pattern "{[local_settings]test_pattern}"
+#    {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_3} --pattern "{[local_settings]test_pattern}"
+#    {envpython} -m unittest discover -v --start-directory {[local_settings]test_path_4} --pattern "{[local_settings]test_pattern}"
```

Comment:
Maybe take this chance to cleanup these unused lines (both here and above and below).

Reply:
Cleaned up in tox.ini